### DISCUSSION TITLE
Support for the  SKR V3 EZ (with STM32H723Vx board)

### DISCUSSION THREAD

**Discussion by vrooze** [mriscoc/Ender3V2S1#917] [General]
I recently acquired this to get a 5th stepper for proper g34 gantry leveling and dual Z.

There are 2 variants of this board (normal and EZ) and two arm H7 chips (743 and 723) turns out I have the latter.

Also chip was only recently added to the bugfix-2.1 branch so I can only compile with a very recent commit of stock Marlin.

This means I can't build the current release of this fw (which is my fw of choice).

Is there any chance this could be added in an upcoming release?
The tip of bugfix-2.1.x has all that's needed.

Thanks!

**Comment by mriscoc** [comment]
You can add the missing files for your board, change the configuration files and try to compile.
